schemas/com.snowplowanalytics.snowplow/media_player_event_percent_progress/jsonschema/1-0-0

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Media player event fired when a percentage boundary set in tracking options is reached",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow",
+    "name": "media_player_event_percent_progress",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "label": {
```

Comment:
% progress property?

Reply:
✅ added `percentProgress` property (removed from the `player` entity schema as it's not necessary).